fedora_manual_setup.md

```diff
@@ -0,0 +1,114 @@
+# Manual setup on Fedora (34)
```

Comment:
```suggestion
# Manual setup on Fedora (36)
```
I got Fedora 36 and this manual is valid

Reply:
I'm trying to replace all instructions with Podman but it's not as simplye as I thought. I will leave this change to someone who actually knows how Podman works, if that's OK.

Reply:
I'll do it myself today. Don't merge this